src/test/java/shop/CartTest.java

```diff
@@ -0,0 +1,75 @@
+package shop;
+
+import com.github.javafaker.Faker;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.Locale;
+
+public class CartTest {
+
+    private static String cartName;
+
+    @BeforeClass
```

Comment:
In case, if you will run tests using maven (f.e. `mvn clean test -Dgroups=addToCart`), then test will fail. It happens, because @Before hook is not in any group. You need to add it in group(s) or add alwaysRun = true attribute.